tests/foreman/ui/test_host.py

```diff
@@ -2343,3 +2343,27 @@ def test_positive_cockpit(session, default_sat):
         ), 'cockpit page shows hostname {} instead of {}'.format(
             hostname_inside_cockpit, default_sat.hostname
         )
+
+
+@pytest.mark.tier4
+def test_positive_read_details_page_from_new_ui(session, module_host_template):
+    """Create new Host and read all its content through details page
+
+    :id: ef0c5942-9049-11ec-8029-98fa9b6ecd5a
+
+    :expectedresults: Host is created and has expected content
+
+    :CaseLevel: System
+    """
+    interface_id = gen_string('alpha')
+    with session:
+        host_name = create_fake_host(session, module_host_template, interface_id, new_ui=True)
+        assert session.host_new.search(host_name)[0]['Name'] == host_name
+        values = session.host_new.get_details(host_name)
+        assert values['Overview']['HostStatusCard']['status'] == 'All Statuses are OK'
+        assert (
+            values['Overview']['DetailsCard']['details']['mac_address'] == module_host_template.mac
+        )
+        assert values['Overview']['DetailsCard']['details']['host_owner'] == values['current_user']
+        # BZ 2059298
+        # assert values['Overview']['DetailsCard']['details']['comment'] == 'Host with fake data'
```

Comment:
So are you waiting for changes before you un-comment this line?

Reply:
Good question, I don't know where to bring this, foreman developers are not creating the HTML structure, they are just using the Patterfly objects, but the structure itself is missing span element, so 3 things in my mind:
1. ask widgetastic developers for help
2. find the workaround in our code ( I was not able to, but maybe you can have an idea :)  ) 
3. this problem is  skipped. Just archived here, in BZ and also in our code,  right now this only occurs for comment, not sure if there can be another field in  `['Overview']['DetailsCard']['details']`

Reply:
I can look into it. Maybe @latran can provide some insight here as well?

Reply:
If you want to, I can give you introduction to the problem :+1: (we can schedule a short call)

Reply:
@ColeHiggins2  I build a reader to read that, it just a mistake that element is missing, right now DEV's looked into that.